rmw_dds_common_generators/rmw_dds_common_generators-extras.cmake.in

```diff
@@ -3,10 +3,8 @@
 
 find_package(ament_cmake_core REQUIRED)
 ament_index_get_resources(rosidl_typesupport_cpp_packages "rosidl_typesupport_cpp")
-ament_index_get_resources(rosidl_generator_packages "rosidl_generator_packages")
 set(_exported_dependencies
   ${rosidl_typesupport_cpp_packages}
-  ${rosidl_generator_packages}
```

Comment:
@ivanpauno just to be clear, you've essentially moved dependency declarations into the CMakeLists.txt using `ament_export_dependencies()` only on the ones you actually need?

Reply:
Exactly.
Before, this was looking for all available generators and exporting them.